.gitignore

```diff
@@ -1,2 +1,4 @@
 /node_modules
 /*.log
+.idea/
+*.iml
```

Comment:
These are workstation-specific files that should be in your [global `.gitignore`](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore).